mmdet3d/models/backbones/minkunet_backbone.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Sequence
+
+from mmengine.model import BaseModule
+from mmengine.registry import MODELS
+from torch import Tensor, nn
+
+from mmdet3d.models.layers import (IS_TORCHSPARSE_AVAILABLE,
+                                   TorchsparseConvModule,
+                                   TorchsparseResidualBlock)
+from mmdet3d.utils import OptMultiConfig
+
+if IS_TORCHSPARSE_AVAILABLE:
+    import torchsparse
+    from torchsparse.tensor import SparseTensor
+else:
+    SparseTensor = None
+
+
+@MODELS.register_module()
+class MinkUNetBackbone(BaseModule):
+    """MinkUNet backbone is the implementation of `4D Spatio-Temporal ConvNets.
+
+    <https://arxiv.org/abs/1904.08755>` with torchsparse backend.
```

Comment:
`_

Reply:
Please take the format of [this line](https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/mmdet3d/models/detectors/fcos_mono3d.py#L14) as the reference.